styles.css

```diff
@@ -385,13 +385,101 @@ h3 {
   margin-bottom: 8px;
 }
 
-#texts {
-  width: 90%;
-  margin: auto;
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  top: 485px;
+  margin-top: 15px;
+}
+
```

Comment:
You are doing great so far but there are a few changes that you've left out to complete your design

[ Required Changes ]

- [ ] Please add the background image for the contact section as it's in the Figma design. It's highlighted in a green arrow in the comment photo below. Please fix that 👍 
- [ ] Kindly add the remaining social media icons that are in the Figma design.
- Spoiler Alert: You will be creating an account for all those social media icons as you go in the course program and hence, it's good you have them now in your portfolio app as you'll update their links once you create the accounts. 👍
Your Design
![image](https://user-images.githubusercontent.com/50941074/159592447-c5d0e1ca-4765-4a1d-83b0-6113dffe9bab.png)

The Figma Design
![image](https://user-images.githubusercontent.com/50941074/159592629-a2c38572-7ed2-440d-8376-7b14dc413dad.png)